xcdat/bounds.py

```diff
@@ -1037,6 +1043,14 @@ def get_bounds_dim(da_bounds: xr.DataArray) -> str:
     This function is designed to work with DataArrays that include a dimension
     representing bounds, which is commonly used in geospatial or temporal data.
     """
+    # If only one bounds dimension is found, return it.
+    bounds_dim = [dim for dim in da_bounds.dims if dim not in da_coords.dims]
+
+    if len(bounds_dim) == 1:
+        return str(bounds_dim[0])
```

Comment:
```suggestion
    if len(bounds_dim) == 1:
        # get dim name
        bdim = str(bounds_dim[0])
        # check if it has len(2)
        if len(da_bounds[bdim]) == 2:
            return bdim
```

Should we also enforce that the `bounds_dim` has len(2)?